cc_utilities/redcap_sync.py

```diff
@@ -122,6 +126,33 @@ def normalize_phone_cols(df, phone_cols):
     return df
 
 
+def populate_symptom_columns(df):
+    """
+    Calculate the 'symptom_count' value and then use it to determine the value
+    of the 'symptomatic' column.
+    """
+    df = df.copy()
+
+    logger.info(f"Calculating and adding column {SYMPTOM_COUNT} to df.")
+    # Count values from column names starting with 'symptoms_selected___'
+    # the checkbox column values will be 1 if selected, 0 if not selected, so
+    # we can just take a sum of the values.
+    symptom_columns = [col for col in df if col.startswith(SYMPTOM_COLUMNS_PREFIX)]
+    symptom_columns.remove(SYMPTOMS_NONE)
```

Comment:
On the off chance `SYMPTOMS_NONE` doesn't exist:
```suggestion
    if SYMPTOMS_NONE in symptom_columns:
        symptom_columns.remove(SYMPTOMS_NONE)
```

Reply:
Good call.